include/qdmi/client.h

```diff
@@ -574,6 +577,77 @@ int QDMI_device_query_operation_property(
     const QDMI_Site *sites, size_t num_params, const double *params,
     QDMI_Operation_Property prop, size_t size, void *value, size_t *size_ret);
 
+/**
+ * @brief Query an telemetry sensor property.
+ * @param[in] device The device to query. Must not be @c NULL.
+ * @param[in] telemetry_sensor The telemetry sensor to query. Must not be @c
+ NULL.
+ * @param[in] prop The property to query. Must be one of the values specified
+ * for @ref QDMI_TelemetrySensor_Property.
+ * @param[in] size The size of the memory pointed to by @p value in bytes. Must
+ * be greater or equal to the size of the return type specified for @p prop,
+ * except when @p value is @c NULL, in which case it is ignored.
+ * @param[out] value A pointer to the memory location where the value of the
+ * property will be stored. If this is @c NULL, it is ignored.
+ * @param[out] size_ret The actual size of the data being queried in bytes. If
+ * this is @c NULL, it is ignored.
+ * @return @ref QDMI_SUCCESS if the device supports the specified property and,
+ * when @p value is not @c NULL, the property was successfully retrieved.
+ * @return @ref QDMI_ERROR_NOTSUPPORTED if the device does not support the
+ * property.
+ * @return @ref QDMI_ERROR_INVALIDARGUMENT if
+ *  - @p device or @p telemetry_sensor is @c NULL,
+ *  - @p prop is invalid, or
+ *  - @p value is not @c NULL and @p size is less than the size of the data
+ *    being queried.
+ * @return @ref QDMI_ERROR_FATAL if an unexpected error occurred.
+ *
+ * @note By calling this function with @p value set to @c NULL, the function can
+ * be used to check if the telemetry sensor supports the specified property
+ without
+ * retrieving the property and without the need to provide a buffer for it.
+ * Additionally, the size of the buffer needed to retrieve the property is
+ * returned in @p size_ret if @p size_ret is not @c NULL.
+ *
+ * @note For example, to query the unit of an telemetry sensor, the following
```

Comment:
```suggestion
 * @note For example, to query the unit of a telemetry sensor, the following
```
As mentioned above, please also check the rest.

Reply:
Corrected